mozetl/hardware_report/hardware_dashboard.py

```diff
@@ -0,0 +1,47 @@
+import click
+from mozetl.hardware_report.summarize_json import *
+from click_datetime import Datetime
+from datetime import datetime, timedelta
+
+
+@click.command()
+@click.option('--start_date',
+              type=Datetime(format='%Y/%m/%d'),
+              default=None,
+              help='Start date, only use this when backfilling (e.g. yyyy/mm/dd)')
+@click.option('--end_date',
+              type=Datetime(format='%Y/%m/%d'),
+              default=None,
+              help='End date, only use this when backfilling (e.g. yyyy/mm/dd)')
+def main(start_date, end_date):
+    if start_date:
+        end_date = start_date + timedelta(days=7)
+
+    # Generate the hardware report
+    start_date = None  # Only use this when backfilling, e.g. dt.date(2016,2,1)
+    end_date = None  # Only use this when backfilling, e.g. dt.date(2016,3,26)
+
+    # Generate the report for the desired period.
+    generate_report(start_date, end_date)
+    # Fetch the previous data from S3 and save it locally.
+    fetch_previous_state("hwsurvey-weekly.json", "hwsurvey-weekly-prev.json")
+    # Concat the json files into the output.
+    print "Joining JSON files..."
+    get_ipython().system(u'jq -s "[.[]|.[]]" *.json > "hwsurvey-weekly.json"')
```

Comment:
Since we've moved out of ipython land, can you change this to use some other method of merging the json?